frontend/src/components/EditAndReviewTable.jsx

```diff
@@ -0,0 +1,109 @@
+import PropTypes from "prop-types";
+import { Link, useNavigate } from "react-router-dom";
+import classNames from "classnames"; // Optional utility library for handling class names
+import { faSort, faSortUp, faSortDown } from "@fa-kit/icons/classic/solid";
+import { FontAwesomeIcon } from "@fortawesome/react-fontawesome";
+
+function TableRow(park) {
+  const navigate = useNavigate();
+
+  function getParkLink() {
+    return `/park/${park.orcs}`;
+  }
+
+  // navigate to park details page
+  function navigateToPark() {
+    navigate(getParkLink());
+  }
+
+  return (
+    <tr onClick={navigateToPark} role="button">
+      <th scope="row">
+        <Link to={getParkLink()}>{park.name}</Link>
+      </th>
```

Comment:
A park name seems to be `#2D2D2D` in the mock.

Reply:
I wrapped it in a link tag and then the color changed (and added an underline). I think it _should_ have some indication that you can click on it, but I'll check with design and see if we should change the color or style

Reply:
I talked to Amanda today, we're thinking the link hover color in the bootstrap theme is incorrect (according to the Design Tokens page) - I'll make a small ticket to override it and fix it site-wide.
CMS-468 - I'll grab it in the next sprint

Reply:
We also decided to add an icon to indicate that the row is clickable, so I might remove the Link tag. I'll make that update along with the loading animation.